packages/vite/src/module-runner/runner.ts

```diff
@@ -55,7 +55,10 @@ export class ModuleRunner {
   private readonly transport: RunnerTransport
   private readonly resetSourceMapSupport?: () => void
   private readonly root: string
-  private readonly moduleInfoCache = new Map<string, Promise<ModuleCache>>()
+  private readonly moduleInfoCache = new Map<
```

Comment:
Is `moduleInfo` the right name for this? It seems a bit confusing as `ModuleInfo` is a type that has a different meaning elsewhere.

Reply:
Changed to `concurrentModuleNodePromises`